CSC322-Grinnell/mayflower-directory
# Staffs & Departments Image Upload

# Changes

* Create new staff + Upload avatar
* Edit staff
* Add department to staff upon creation/edit
* Listings of all staffs
* Listing of staffs under departments

# Testing
1. Run `rails db:migrate`
2. Go to staff page
3. Try create new staff + upload avatar + Add one/multiple deparment to that staff
4. Check if the new staff shows up under staff listing.
5. Check if the staff shows up under department they are in. 
6. Edit staff
7. Try removing staff from one department and check whether that staff stops showing up under that department.
